packages/bpk-docs/src/pages/MobileScrollContainerPage/MobileScrollContainerPage.js

```diff
@@ -42,10 +44,11 @@ const MobileScrollContainerPage = () => (
         viewport and container fluidity in mind i.e. horizontal list items
         should stack on top of each other and overflowing text should wrap on to
         the next line. There are some cases where it is desireable to maintain a
-        fixed width on certain content (see bar charts and horizontal
-        navigation). Mobile scroll containers will hide overflowing content and
-        display a fading indicator to let the user know that the content is
-        scrollable.
+        fixed width on certain content (see{' '}
```

Comment:
`{' '}` is this just functioning as `&nbsp;`?

Reply:
Yeah! I didn't add it - prettier did 😏

Reply:
Ahh ok!